### DISCUSSION TITLE
Setting to disable metadata read out

### DISCUSSION THREAD

**Discussion by beniw** [directus/directus#9356] [Feature Requests]
Hello everyone

I like to place a feature request for the handling of metadata on images. It would be nice if we could disable the metadata read out on files in the settings.

We have people who get confused about the pre-filled tags and the description and would rather not have it there.

Also we recently experienced some issues with weird exif/metadata on images which only happen in certain cases. Like on a local machine the upload works fine but on the server installation the same image crashes the directus container. The upload of the same image without metadata works perfectly fine on the server.

I first tried to fix this over the files.upload hook but as there is no before modifier this did not really work out for me.

kind regards

**Comment by rijkvanzanten** [comment]
What would be the ideal way to configure this? I reckon you might want to be able to pick and choose what meta flags to extract, while disabling others. Maybe a `STORAGE_EXTRACT_META` flag could do it, where it would allow a CSV of directus_files fields to try to extract. `STORAGE_EXTRACT_META="*"` would be the default (that's the current behavior), and then you could set it to `STORAGE_EXTRACT_META="tags,title"` to extract specific values, and `STORAGE_EXTRACT_META="false"` to disable it completely 🤔

**Reply by beniw** [reply]
This is actually a really smart idea. 
I was thinking about a flag to disable them all together but to make them configurable would be even better. 👍

**Reply by rijkvanzanten** [reply]
Wanna open a PR for this change? I think it might be as simple as adding an additional check against that env var in these three blocks 🤔 

https://github.com/directus/directus/blob/6ea29d86736133947a2a43b1a107d457faedc826/api/src/services/files.ts#L99-L107

**Reply by beniw** [reply]
I will try. Not sure when I'll have time for it but it sure would be nice to contribute 😃

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
